README.md

```diff
@@ -1,2 +1,15 @@
 # code-genie-bot
 telegram bot for code genie
+
+## Team Members
+Basel Amin\
+Mohammad Shaheen\
+Hasan Masalha\
+Aya Abbas\
+Shaden Hakim
+## Product Goal
+a telegram bot for gode genie
+
+## General Architecture
+bot for code genie \
```

Comment:
this is not a pr state.
if its not final - dont open pr

plus, go over the issue and see that there are some detials missing

Reply:
I updated it, I believe its ok now